dali/base/dafdesc.cpp

```diff
@@ -1505,6 +1531,11 @@ class CFileDescriptor:  public CFileDescriptorBase, implements ISuperFileDescrip
         IPropertyTree *t = &queryProperties();
         if (!isEmptyPTree(t))
             pt.addPropTree("Attr",createPTreeFromIPT(t));
+        t = &queryHistory();
+        if (!isEmptyPTree(t))
```

Comment:
See other comments, implies not safe/ potential illegal use of reference to null